examples/basic/data/color_mappers.py

```diff
@@ -2,11 +2,11 @@
 log mapping and linear mapping with different color palette.
 
 .. bokeh-example-metadata::
-    :apis: bokeh.plotting.Figure.scatter, bokeh.models.ColumnDataSource, bokeh.models.annotations.ColorBar, bokeh.models.mappers.LinearColorMapper, bokeh.models.mappers.LogColorMapper # noqa: E501
+    :apis: bokeh.plotting.Figure.scatter, bokeh.models.ColumnDataSource, bokeh.models.annotations.ColorBar, bokeh.models.mappers.LinearColorMapper, bokeh.models.mappers.LogColorMapper
     :refs: :ref:`ug_topics_images_colormapped`
     :keywords: color, tools, scatter, data_map
 
-'''
+''' # noqa: E501
```

Comment:
Some day we will get automatic api-linking in example code and do away with `:apis:` alogether, but until then, this is much nicer to work with long docstring lines.